sphinx_rtd_theme/__init__.py

```diff
@@ -31,6 +32,12 @@ def config_initiated(app, config):
             _('The canonical_url option is deprecated, use the html_baseurl option from Sphinx instead.')
         )
 
+
+def extend_html_context(app, pagename, templatename, context, doctree):
+     # Extend the page context to be albe to parse versions correctly.
+     context['parse_version'] = packaging.version.parse
```

Comment:
```suggestion
     # Add ``sphinx_version_info`` tuple for use in Jinja templates
     context['sphinx_version_info'] = sphinx_version[:3]
```

Reply:
Same mistake as before - you are assuming stuff about the structure of the version string. Is there a guarantee that there will always be 3 numbers? Probably OK, given the history of the sphinx versioning, but you might have said the same about no ``post`` versions until last weekend...

Personally, I think parsing the version and using the supported PEP440 definition of what version comparison should be is the more correct solution. I accept though that it is less "simple" than the one proposed here.

Reply:
It's not a string that is being sliced, it's a tuple:
https://github.com/sphinx-doc/sphinx/blob/5.x/sphinx/__init__.py#L33

Sphinx follows semver, so I trust this value will always be 3 integers, plus additional specifiers. @AA-Turner has more context here, as a maintainer of Sphinx though, so I trust this recommendation.